winml/lib/Api.Ort/OnnxruntimeEnvironment.cpp

```diff
@@ -16,30 +16,38 @@ static bool debug_output_ = false;
 
 EXTERN_C IMAGE_DOS_HEADER __ImageBase;
 
-static bool IsCurrentModuleInSystem32() {
-  std::string current_module_path;
-  current_module_path.reserve(MAX_PATH);
-  auto size_module_path = GetModuleFileNameA((HINSTANCE)&__ImageBase, current_module_path.data(), MAX_PATH);
-  FAIL_FAST_IF(size_module_path == 0);
-
+static std::string System32Path() {
   std::string system32_path;
   system32_path.reserve(MAX_PATH);
   auto size_system32_path = GetSystemDirectoryA(system32_path.data(), MAX_PATH);
```

Comment:
GetSystemDirectoryA handles the scenario where SysWOW64 folder is needed right?

Reply:
I think GetSystemWow64Directory does that... testing this now.

Reply:
The first iteration doesnt work for WOW64.. fixed.

Now trying to resolve WinSxS.

Reply:
ACTUALLY, so LoadLibraryExA("C:\\windows\\system32\\...") will automagically resolve to syswow64 when needed!

Reply:
Nicee!!